modules/custom/d_product/config/install/core.entity_view_display.media.d_image.d_product_image.yml

```diff
@@ -1,13 +1,17 @@
+uuid: a5eec55c-3d55-4c42-a8cd-925da2bbefd4
 langcode: en
 status: true
 dependencies:
   config:
     - core.entity_view_mode.media.d_product_image
     - field.field.media.d_image.field_media_image
+    - image.style.product_thumbnail_pc
     - media.type.d_image
-    - responsive_image.styles.product_thumbnail
   module:
-    - responsive_image
+    - lazy
+    - svg_image
+_core:
+  default_config_hash: ajA2zMt8GeAwTxApzRx6B0krHb8IYGu72bJpu3HEkJk
```

Comment:
Do wywalenia, lepiej eksportować featuresami

Reply:
To zostały jakieś śmieci, miałem to wywalić, najwidoczniej przeoczyłem